app/routes/enterprise.py

```diff
@@ -41,6 +43,15 @@ class EnterpriseUpdateResponse(BaseModel):
         address: Optional[str]
         nip_number: Optional[validators.NipNumber]
 
+    class UserEnterpriseResponse(BaseModel):
+        enterprise_id: int
+        user_id: int
+        role: models.UserEnterpriseRoles
+
+    class UserEnterpriseGrantAccess(BaseModel):
+        user_id: int
+        role_to_grant: str
```

Comment:
Switch role_to_grant to enum type so that we will have enumerated list of possible values (as above)